rook/helmfile.d/values/cluster.yaml.gotmpl

```diff
@@ -105,3 +105,7 @@ cephBlockPools:
 cephFileSystems: []
 
 cephObjectStores: []
+
+cephBlockPoolsVolumeSnapshotClass:
+  enabled: true
+  isDefault: true
```

Comment:
Does UpCloud not have block storage or why do we use Rook/Ceph here?

Reply:
https://github.com/elastisys/compliantkubernetes-kubespray/issues/362#:~:text=Also%20add%20configuration%20for%20adding%20volumeSnapshotClasses%20for%20each%20CSI%20driver.

The issue description asks that we have config for enabling volumesnapshotclass for all CSI drivers.